test/DebugInfo/top_level_code.swift

```diff
@@ -9,6 +9,6 @@ markUsed(a+b)
 // CHECK: _main:
 // Verify that the top-level function (main) begins at line 0 and then
 // proceeds to line 6.
-// CHECK: .loc    {{[0-9]}} 0 {{[0-9]}} prologue_end
+// CHECK: .loc    {{[0-9]}} 0 {{[0-9]}} 
 // CHECK-NOT: .loc
-// CHECK: .loc    {{[0-9]}} 6 {{[0-9]}}
+// CHECK: .loc    {{[0-9]}} 6 {{[0-9]}} prologue_end
```

Comment:
Interestingly enough, this is something that I was recently looking at.  Something has changed and when building swift against upstream-with-swift on clang/LLVM, it seems that the prolgue_end marker is no longer emitted.  The MF does see the FrameSetup markers on the prologue instructions, but when the line record is constructued, we ignore the FrameSetup MIFlag and just drop the prologue tracking on the floor.

Also, it seems that for some reason this test is marked as XFAIL on Linux.  Why is that?